cfep-13.md

```diff
@@ -0,0 +1,70 @@
+
+<table>
+<tr><td> Title </td><td> Securing `conda-forge` Uploads to `anaconda.org` </td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Matt, Marius, Anthony, Isuru</td></tr>
+<tr><td> Created </td><td> Feb 27, 2020</td></tr>
+<tr><td> Updated </td><td> Feb 27, 2020</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Currently the `anaconda.org` tokens on each feedstock allow any `conda-forge`
+feedstock maintainer to upload any output to any `anaconda.org` package.
+This could allow someone to push a compromised and/or malicious build
+of say the `python` package. This situation is a security risk that we
+should mitigate.
+
+## Specification
+
+We will secure the uploads of outputs to `anaconda.org` via the following process.
+
+1. Each feedstock is provisioned with a secret key that is only known to the
+   maintainers of that feedstock and the core team. These keys will be kept in
+   a private repo with only core members having read access. Core members and 
+   encrypted GitHub keys for bot admins will have write access in order to generate 
+   the secret keys and push then to the CI providers.
```

Comment:
Shouldn't this be in keybase rather than on github?

Reply:
We can do this, but IDK how to push and pull from keybase with a key securely when that key is stored on staged-recipes.

Reply:
We may not want keys for keybase itself floating around in our CI scripts.

Reply:
hmm yeah, I see